database/populate.py

```diff
@@ -13,21 +14,45 @@
     'painter4': Painter('Van Gogh4')
 }
 
-paintings = [
-    Painting(painters['painter1'], 'Starry Night1', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
-    Painting(painters['painter2'], 'Starry Night2', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
-    Painting(painters['painter3'], 'Starry Night3', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
-    Painting(painters['painter4'], 'Starry Night4', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
-    Painting(painters['painter4'], 'Starry Night5', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
-    Painting(painters['painter4'], 'Starry Night5', '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg')
+paintings = {
+    'painting1': Painting(painters['painter1'], 'Starry Night1',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
+    'painting2': Painting(painters['painter2'], 'Starry Night2',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
+    'painting3': Painting(painters['painter3'], 'Starry Night3',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
+    'painting4': Painting(painters['painter4'], 'Starry Night4',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
+    'painting5': Painting(painters['painter4'], 'Starry Night5',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg'),
+    'painting6': Painting(painters['painter4'], 'Starry Night5',
+                          '1024px-Van_Gogh_-_Starry_Night_-_Google_Art_Project.jpg')
+}
+
+saints = {
+    'saint1': Saint('Saint George'),
+    'saint2': Saint('Saint Stan'),
+    'saint3': Saint('Saint Anthony'),
+}
+
+
+
+relationship_saints_painting = [
```

Comment:
why not 
saint['saint1'].paintings.extend([Painting(), Painting(), ...])
#see http://stackoverflow.com/questions/252703/python-append-vs-extend